encoderpy/target_encoder.py

```diff
@@ -0,0 +1,47 @@
+def target_encoder(X_train, X_test, y, cat_columns, prior = 0.5, min_samples = 1):
+  """
+  This function encodes categorical variables with average target values for each category.
+  
+  Parameters
+  ----------
+  X_train : pd.DataFrame
+          A pandas dataframe representing the training data set containing some categorical features/columns.
+  X_test : pd.DataFrame
+          A pandas dataframe representing the test set, containing some set of categorical features/columns.
+  y : pd.Series
+          A pandas series representing the target variable. If the objective is "binary", then this
+          series should only contain two unique values.
```

Comment:
Perhaps mention that the positive class be encoded as "1", and the negative class as "0"? Or something along those lines?

Reply:
Resolved.